jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml

```diff
@@ -59,5 +59,11 @@
             job-env: |
                 export E2E_NAME="e2e-kops-aws-updown"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[k8s.io\]\sNetworking.*\[Conformance\]"
+        - 'aws':  # kubernetes-e2e-kops-aws
+            description: 'Run e2e tests on AWS deployed via kops using the latest successful Kubernetes build.'
+            timeout: 240
+            job-env: |
+                export E2E_NAME="e2e-kops-aws"
+                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|NodeProblemDetector"
```

Comment:
This is actually copied over from the AWS tests, but I forgot to include the comment. I'll push that now.